minet/instagram/utils.py

```diff
@@ -0,0 +1,25 @@
+# =============================================================================
+# Minet Instagram Utils
+# =============================================================================
+#
+# Miscellaneous helper functions used by the minet.instagram namespace.
+#
+
+import re
+from datetime import datetime
+
+
+def extract_from_text(text, char):
+    splitter = re.compile(r"[^\w%s]+" % char)
```

Comment:
This regex should be compiled outside of the function to avoid needing to recompile it each time we call the function.